dotnet/src/Connectors/Connectors.MistralAI.UnitTests/Client/MistralClientTests.cs

```diff
@@ -504,6 +522,31 @@ public void ValidateToMistralChatMessages(string roleLabel, string content)
         Assert.Single(messages);
     }
 
+    [Fact]
+    public void ValidateToMistralChatMessagesWithMultipleContents()
+    {
+        // Arrange
+        using var httpClient = new HttpClient();
```

Comment:
Is `httpclient` really necessary to create a MistralClient here, I see no usage of it.
```suggestion
```

Reply:
It's used on the next line. Too much sunshine ;-)

Reply:
I saw that, is that the httpClient is being created without any handler, for that reason, I don't think it is needed, unless the handler was used to inject testing behavior.